atmo/jobs/forms.py

```diff
@@ -221,10 +221,13 @@ def clean_start_date(self):
     def save(self, commit=True):
         obj = super(EditSparkJobForm, self).save(commit=False)
         if 'start_date' in self.changed_data:
```

Comment:
No sorry, I mean with the "TODO" in this file. On `save` currently we reset the `last_run_date` if the `start_date` is edited in the form. With historical job runs this doesn't make sense to me and I would think this should be removed. But I wanted to double check.